docs/source/index.rst

```diff
@@ -67,6 +67,8 @@ mind.
 - PyPI: https://pypi.org/project/xgi/
 - Documentation: https://xgi.readthedocs.io/
 
+Sign up for our `mailing list <http://eepurl.com/igE6ez>`_!
```

Comment:
noice

Reply:
Should we add a link to twitter/mastodon somewhere too?

Reply:
Sure!

Reply:
Done.